src/ui/TypingIndicatorMessage/index.tsx

```diff
@@ -1,48 +1,53 @@
-import React, {ReactElement, useContext, useRef} from 'react';
-import './index.scss';
-import {getClassName} from '../../utils';
+import React, { ReactElement, useContext, useRef } from 'react';
+import { Member } from '@sendbird/chat/groupChannel';
+import { LocalizationContext } from '../../lib/LocalizationContext';
 import Avatar from '../Avatar';
-import {Member} from '@sendbird/chat/groupChannel';
-// @ts-ignore
-import typingIndicatorLogo from '../../gifs/message-typing-indicator.gif';
-import {LocalizationContext} from '../../lib/LocalizationContext';
-
-const AVATAR_SIZE = '28px';
+import Label, { LabelColors, LabelTypography } from '../Label';
 
 export interface TypingIndicatorMessageProps {
-  className?: string | Array<string>;
   typingMember: Member;
 }
 
 export default function TypingIndicatorMessage({
-  className,
   typingMember,
 }: TypingIndicatorMessageProps): ReactElement {
-  const avatarRef = useRef(null);
   const { stringSet } = useContext(LocalizationContext);
+  const avatarRef = useRef(null);
```

Comment:
```suggestion
  const avatarRef = useRef<HTMLDivElement>(null);
```

Reply:
But do we need this? I can't see it being used anywhere

Reply:
i guess we dont need i removed it